crmsh/bootstrap.py

```diff
@@ -1625,8 +1625,6 @@ def join_ssh_impl(local_user, seed_host, seed_user, ssh_public_keys: typing.List
     user_by_host.add(local_user, utils.this_node())
     user_by_host.set_no_generating_ssh_key(bool(ssh_public_keys))
     user_by_host.save_local()
-    user_by_host.add(seed_user, get_node_canonical_hostname(seed_host))
```

Comment:
This is required by line 1633 when a hostname alias is used.

Reply:
Then we can try to check init node's cluster service status just after the line 1620

Reply:
Then the process/order for this part will be:
1. call ping_node to check if the init node is up
2. call join_ssh to setup passwordless with init node
3. inside join_ssh, after swap the key for the first user('hacluster' is the second), and before using utils.HostUserConfig to save the info of user, check the status of cluster service on init node
4. then get the canonical hostname from init node